ompi/include/mpi.h.in

```diff
@@ -406,28 +404,16 @@ typedef int (MPI_Grequest_query_function)(void *, MPI_Status *);
 typedef int (MPI_Grequest_free_function)(void *);
 typedef int (MPI_Grequest_cancel_function)(void *, int);
 
-#if !OMPI_OMIT_MPI1_COMPAT_DECLS
 /*
- * Removed typedefs. These will be deleted in a future Open MPI release.
+ * Deprecated typedefs. These will be deleted in a future MPI standard.
```

Comment:
I said this on the last PR: "deprecated" != "will be deleted in a future MPI standard".

Reply:
Dang it, I missed this, thanks.

Any idea why the build Checker failed?  Something about --no-orte and --no-oshmem failures, but that's all I can understand.

Reply:
Looks like a local AWS failure.  Just do a retest.

Reply:
Hmm, so that's an AWS filesystem failure?   I hope so, since I'm not sure how it generates that orte/constants.h (which is what it couldn't find).

Reply:
Hmm when I looked at the logs for those 2 builds (admittedly somewhat quickly), I thought they said they had problems clearing out the working directory to do the next build.  I took that to mean it was a local AWS error.  Perhaps I read them wrong / too quickly...?

Regardless, you just pushed to the PR again -- we'll see how AWS CI does this time...